pkgs/development/tools/language-servers/fennel-ls/default.nix

```
@@ -0,0 +1,23 @@
+{ lib, stdenv, fetchgit, lua, luaPackages }:
+stdenv.mkDerivation rec {
+  pname = "fennel-ls";
+  version = "0.1.0";
+
+  src = fetchgit {
+    url = "https://git.sr.ht/~xerool/fennel-ls/";
+    rev = version;
+    sha256 = "sha256-RW3WFJGwascD4YnnrAm/2LFnVigzgtfzVubLMDW9J5s=";
+  };
+
+  buildInputs = [ lua luaPackages.fennel];
+  makeFlags = [ "PREFIX=$(out)" ];
+  installFlags = [ "PREFIX=$(out)" ];
+
+  meta = with lib; {
+    description = "fennel language server";
+    homepage = "https://git.sr.ht/~xerool/fennel-ls/";
+    license = licenses.mit;
```

I think the package needs a maintainer before it can be accepted. Would you like to maintain it? (you'll need to add yourself to `maintainers/maintainer-list.nix` as well)

```suggestion
    license = licenses.mit;
    maintainers = with maintainers; [ yisraeldov ];
```